samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -642,7 +642,7 @@ static int connect_cloud(void)
  *
  * @return int - 0 on success, otherwise a negative error code.
  */
-static int setup_lte(void)
+static int setup_modem(void)
```

Comment:
if FMFU is enabled and a delta update is performed, the code below will print out an invalid error message.
i think changing the check to `if (err < 0)` should be sufficient.
For reference, see https://github.com/nrfconnect/sdk-nrf/blob/052c17e36db62b5093538d0dc1be8ed0dab475f3/applications/asset_tracker_v2/src/main.c#L235

Reply:
Good catch -- I've decided to add debug logging for all three possibilities